Package.swift

```diff
@@ -604,7 +604,7 @@ let package = Package(
             url: "https://github.com/apple/swift-openapi-urlsession", .upToNextMajor(from: "1.0.2")
         ),
         .package(url: "https://github.com/tuist/Path", .upToNextMajor(from: "0.3.0")),
-        .package(url: "https://github.com/tuist/XcodeGraph", .upToNextMajor(from: "1.23.0")),
+        .package(url: "https://github.com/tuist/XcodeGraph", branch: "fix/package-info-default-isolation"),
```

Comment:
We need to merge [this one first](https://github.com/tuist/XcodeGraph/pull/326/)